diode-react/src/main/scala/diode/react/ReactConnector.scala

```diff
@@ -116,18 +116,19 @@ trait ReactConnector[M <: AnyRef] { circuit: Circuit[M] =>
       }
 
       private def changeHandler(cursor: ModelRO[S]): Unit = {
-        val isMounted = t.isMounted
+        val isMounted = t.isMounted.map(_.getOrElse(false))
```

Comment:
Shouldn't this be `_.getOrElse(true)`? Maybe I am misunderstanding the semantics, but IMHO we rather want to have an unnecessary `updateState` call than missing one, right?